cli/package.json

```diff
@@ -25,7 +25,9 @@
     "ajv": "^8.12.0",
     "commander": "^12.0.0",
```

Comment:
Are we removing Mermaid now that it's no longer needed? That should remove the dependency on Playwright/puppeteer

Reply:
yeah good point I forgot to remove unused deps